lib/ssl/src/ssl.erl

```diff
@@ -121,6 +121,12 @@ information about the supported standards see [ssl(6)](ssl_app.md).
 %% Tracing
 -export([handle_trace/3]).
 
+-deprecated([{prf,5,"Use export_key_materials/4 instead. "
+              "Note that in OTP 28 the 'testing' way of calling this function will no longer be supported."
+              }]).
+
+-deprecated_type([{prf_random, 0,"Only used in deprecated function prf/5 will no longer bee needed."}]).
```

Comment:
typo: bee -> be (check it globally pls, looks like copy/pasted)

Reply:
fixed